packages/amplify-graphql-api-construct-tests/src/__tests__/amplify-ddb-canary.test.ts

```diff
@@ -0,0 +1,33 @@
+import * as path from 'path';
+import { createNewProjectDir, deleteProjectDir } from 'amplify-category-api-e2e-core';
+import { cdkDestroy } from '../commands';
+import { setupBackend, testGraphQLOperations } from '../canary-tests-common';
+
+jest.setTimeout(1000 * 60 * 60 /* 1 hour */);
+
+describe('Canary using Amplify DynamoDB model datasource strategy', () => {
+  let projRoot: string;
+  let projFolderName: string;
+
+  beforeEach(async () => {
+    projFolderName = 'amplifyddbcanary';
+    projRoot = await createNewProjectDir(projFolderName);
+  });
+
+  afterEach(async () => {
+    try {
+      await cdkDestroy(projRoot, '--all');
+    } catch (_) {
+      /* No-op */
```

Comment:
Why are we swallowing destroy errors? I'm not (yet) saying we should fail the test, but we should probably know about the failures right?

Reply:
If there are any errors during destroy, they are captured in the command execution playbacks that we store as artifacts.